.travis.sh

```diff
@@ -11,4 +11,7 @@ else
     git submodule update --init --recursive
     make test
     make lint
+    git clone https://www.github.com/dlang/phobos.git --depth=1
+    # just check that it doesn't crash
+    cd phobos && ../dscanner -S --config=.dscanner.ini || true
```

Comment:
I'm not sure that it's a good idea to keep this but at least it shows that the segmentation fault is gone.

Reply:
pro: This can be useful when new checks are added, in order to detect false positives and see a bit what will happen when used IRL.